src/styles/utils/container.scss

```diff
@@ -0,0 +1,17 @@
+.container {
+  width: 100%;
+
+  &__card {
+    --count-columns: 1;
+
+    box-sizing: border-box;
+    display: grid;
+    gap: 46px 48px;
+    justify-content: center;
+    margin-block: 50px;
+    margin-inline: 40px;
+    grid-template-columns: repeat(var(--count-columns), 200px);
+
+    @include m.change-size-display(2, 3, 4);
```

Comment:
Ensure that the mixin `m.change-size-display` is correctly defined in the `mixins` file and that it is used to handle responsive design changes appropriately.